EditorInterface.cpp

```diff
@@ -2997,16 +3254,19 @@ void EditorInterface::IGSquadEditor()
 						}
 					}
 					ImGui::BeginChild("ChoreoSlots", ImVec2(0, 0), true);
-					for (auto &slot : ckey->slots) {
-						ImGui::PushID(&slot);
-						ImGui::DragFloat3("Position", &slot.position.x, 0.1f);
-						ImGui::DragFloat3("Direction", &slot.direction.x, 0.1f);
-						ImGui::InputScalar("Enemy pool", ImGuiDataType_S8, &slot.enemyGroup);
-						ImGui::PopID();
-						ImGui::Separator();
+					if (IsDoingHomeWork == false) {
```

Comment:
I did look into to the issue, but i eventually settled for this because i couldn't figure out what was going wrong exactly. Maybe you could help me out there 😸